pkgs/development/tools/imatix_gsl/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchgit, pcre } :
+
+stdenv.mkDerivation rec {
+    name = "imatix_gsl";
+    version = "4.1";
+    src = fetchgit {
+        url = "https://github.com/imatix/gsl.git";
+	rev = "72192d0d9de17de08d9379602d6482b4e5d402d0";
+	sha256 = "0dyginj8l8f5rgbhlp4f3lhfz2ixdli9192l1bmc5jviklna4phg";
+    };
+
+    buildInputs = [ pcre ];
+
+    buildPhase = ''
+    	cd src
+	make
+    '';
+
+    installPhase = ''
+    	make install DESTDIR=$out
+    '';
+
+    meta.description = ''
+    	Code construction tool developed by iMatix
```

Comment:
Bikeshedding: the "developed by iMatix" part is redundant, as a potential user it does not help me decide whether I need this.